pandas/core/arrays/categorical.py

```diff
@@ -1601,6 +1601,9 @@ def sort_values(self, inplace=False, ascending=True, na_position="last"):
         na_position : {'first', 'last'} (optional, default='last')
             'first' puts NaNs at the beginning
             'last' puts NaNs at the end
+        key : Callable, default None
+            If not None, apply the key function to every value before
+            sorting. Identical to key argument in built-in sorted function.
```

Comment:
add versionadded 1.0.0

Reply:
@ja3067 can you add this. `.. versionadded:: 1.0.0`.

Reply:
added